extras/nginx_config_model.txt
sudo fuser -k 8081/tcp

<form class="success" id="form-id" action="{% url "formly_dt_page_create" pk=selected_survey.pk %}" method="post">
    {% csrf_token %}
    <a class="nav-link {% if page == selected_page %}active{% endif %}" onclick="document.getElementById('form-id').submit();">Add page</a>
</form>


project's root directory with a command similar to this:
gunicorn -c gunicorn/gunicorn-config.py your-project.wsgi
gunicorn -c gunicorn/gunicorn-config.py core.wsgi
===================================================

#user  nginx;
worker_processes  1;

error_log  /var/log/nginx/error.log;
error_log  /var/log/nginx/error.log  notice;
error_log  /var/log/nginx/error.log  info;

#pid        /var/run/nginx.pid;

include /etc/nginx/modules.conf.d/*.conf;

events {
    worker_connections  1024;
}


http {
    include       mime.types;
    default_type  application/octet-stream;

    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';

    access_log  /var/log/nginx/access.log  main;

    sendfile        on;
    #tcp_nopush     on;

    #keepalive_timeout  0;
    keepalive_timeout  65;
    #tcp_nodelay        on;

    #gzip  on;
    #gzip_disable "MSIE [1-6]\.(?!.*SV1)";

    server_tokens off;

    include /etc/nginx/conf.d/*.conf;
}

# override global parameters e.g. worker_rlimit_nofile
include /etc/nginx/*global_params;
It worked



=========================================================

upstream merakimvsense-gunicorn {
	server unix:/tmp/gunicorn.sock fail_timeout=0;
}

server {

	listen 80 default;
	client_max_body_size 4G;
	server_name your-site.com;
	keepalive_timeout 70;

	access_log /var/log/nginx/access_log;
	error_log /var/log/nginx/error_log;

	root /opt/merakimvsense/core;

	location /static/ {
		autoindex on;
		alias /opt/merakimvsense/core/static/;
		expires 1M;
		access_log off;
		add_header Cache-Control "public";
		proxy_ignore_headers "Set-Cookie";
	}

	location @proxy_merakimvsense {
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
		proxy_set_header Host $http_host;
		proxy_redirect off;
		proxy_pass merakimvsense-gunicorn;
	}

	location / {
		try_files $uri @proxy_merakimvsense;
	}
}


=======================================
working
=======================================
upstream merakimvsense-gunicorn {
	server unix:/tmp/gunicorn.sock fail_timeout=0;
}

server {

	listen 80 default;
	client_max_body_size 4G;
	server_name your-site.com;
	keepalive_timeout 70;

	access_log /var/log/nginx/access_log;
	error_log /var/log/nginx/error_log;

	root /opt/merakimvsense/core;

	location /static/ {
		autoindex on;
		alias /opt/merakimvsense/core/static/;
		expires 1M;
		access_log off;
		add_header Cache-Control "public";
		proxy_ignore_headers "Set-Cookie";
	}

	location @proxy_merakimvsense {
		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
		proxy_set_header Host $http_host;
		proxy_redirect off;
		proxy_pass http://merakimvsense-gunicorn;
	}

	location / {
		try_files $uri @proxy_merakimvsense;
	}
}


core/settings.py
# -*- encoding: utf-8 -*-
"""
License: MIT
Copyright (c) 2019 - present AppSeed.us
"""

import os
from decouple import config
from unipath import Path
import dj_database_url

# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
BASE_DIR    = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
PROJECT_DIR = Path(__file__).parent

# SECURITY WARNING: keep the secret key used in production secret!
SECRET_KEY = config('SECRET_KEY', default='S#perS3crEt_1122')

# SECURITY WARNING: don't run with debug turned on in production!
#DEBUG = config('DEBUG', default=False)
DEBUG = config('DEBUG', default=True)


# load production server from .env
ALLOWED_HOSTS = ['localhost', '127.0.0.1', '5.196.27.225', config('SERVER', default='127.0.0.1')]

CORS_ORIGIN_ALLOW_ALL = False
CORS_ORIGIN_WHITELIST = (
    'http://localhost:8081',
)

# Application definition

INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'rest_framework',
    'corsheaders',
    'app',
    'cliente.apps.ClienteConfig',
    'instalacion.apps.InstalacionConfig',
    'camara_zona.apps.CamaraZonaConfig',
    'monitor.apps.MonitorConfig',
]

MIDDLEWARE = [
    'django.middleware.security.SecurityMiddleware',
    'django.contrib.sessions.middleware.SessionMiddleware',
    'django.middleware.common.CommonMiddleware',
    'django.middleware.csrf.CsrfViewMiddleware',
    'django.contrib.auth.middleware.AuthenticationMiddleware',
    'django.contrib.messages.middleware.MessageMiddleware',
    'django.middleware.clickjacking.XFrameOptionsMiddleware',
    'whitenoise.middleware.WhiteNoiseMiddleware',
]

ROOT_URLCONF = 'core.urls'
LOGIN_REDIRECT_URL = "home"   # Route defined in app/urls.py
LOGOUT_REDIRECT_URL = "home"  # Route defined in app/urls.py
TEMPLATE_DIR = os.path.join(BASE_DIR, "core/templates")  # ROOT dir for templates

TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [TEMPLATE_DIR],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
                'instalacion.context_processors.side_menu_instalaciones',
            ],
        },
    },
]



WSGI_APPLICATION = 'core.wsgi.application'

# Database
# https://docs.djangoproject.com/en/3.0/ref/settings/#databases


# DATABASES = {
#         'default': {
#             'ENGINE': 'djongo',
#             'NAME': 'MerakiMVSenseDBSeguridad',
#             'ENFORCE_SCHEMA': False,
#             'CLIENT': {
#                 'host': 'mongodb://smfactory:S0ftwar4Mf4ac0ory@127.0.0.1:27117',
#                 'port': 27117,
#                 'username': 'smfactory',
#                 'password': 'S0ftwar4Mf4ac0ory',
#                 'authSource': 'admin',
#                 'authMechanism': 'SCRAM-SHA-1'
#             }
#        }
#

DATABASES = {
    'default': {
        'ENGINE': 'djongo',
        'NAME': 'SMF_BDControlAforo',
        'HOST': '127.0.0.1',
        'PORT': 27017,
    },
    'mosayk': {
        'ENGINE': 'djongo',
        'NAME': 'SMF_BDMosayk',
        'HOST': '127.0.0.1',
        'PORT': 27017,
    }
}

# Password validation
# https://docs.djangoproject.com/en/3.0/ref/settings/#auth-password-validators

AUTH_PASSWORD_VALIDATORS = [
    {
        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
    },
    {
        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
    },
    {
        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
    },
    {
        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
    },
]


# Internationalization
# https://docs.djangoproject.com/en/3.0/topics/i18n/

LANGUAGE_CODE = 'en-us'

TIME_ZONE = 'UTC'

USE_I18N = True

USE_L10N = True

USE_TZ = True

#############################################################
# SRC: https://devcenter.heroku.com/articles/django-assets

BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))

# Static files (CSS, JavaScript, Images)
# https://docs.djangoproject.com/en/1.9/howto/static-files/
STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
STATIC_URL = '/static/'

MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
MEDIA_URL = '/media/'


# Extra places for collectstatic to find static files.
STATICFILES_DIRS = (
    os.path.join(BASE_DIR, 'core/static'),
)


#Allow Log by email or console
ADMINS = (('Ligia', 'ligiapuertas@gmail.com'), ('Joel', 'joelsegoviacrespo@gmail.com'))
LOGGING = {
    'version': 1,
    'disable_existing_loggers': True,
    'formatters': {
        'verbose': {
            'format': '%(asctime)s %(levelname)s [%(name)s:%(lineno)s] %(module)s %(process)d %(thread)d %(message)s'
        }
    },
    'handlers': {
        'gunicorn': {
            'level': 'DEBUG',
            'class': 'logging.handlers.RotatingFileHandler',
            'formatter': 'verbose',
            'filename': '/opt/gunicorn.errors',
            'maxBytes': 1024 * 1024 * 100,  # 100 mb
        }
    },
    'loggers': {
        'gunicorn.errors': {
            'level': 'DEBUG',
            'handlers': ['gunicorn'],
            'propagate': True,
        },
    }
}